app/src/main/java/com/example/dispatcher/presentation/favorites/viewModel/FavoritesViewModel.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.dispatcher.presentation.favorites.viewModel
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import com.example.dispatcher.presentation.homepage.model.Article
+import com.example.dispatcher.domain.homepage.repo.ArticleRepository
+
+class FavoritesViewModel(application: Application) : AndroidViewModel(application) {
+
+    private val articleRepository = ArticleRepository(application)
+    private lateinit var articlesList: MutableList<Article>
```

Comment:
this is incorrect use of 'lateinit' as this variable initialize inside the `init {}` scope, so it equal to:     `private var articlesList: MutableList<Article> = articleRepository.getArticles().toMutableList()`
lateinit is when you'll init the variable outside of the inline declaration or the `init{}` scope.